modules/rampart-trust/src/main/java/org/apache/rahas/impl/SAML2TokenIssuer.java

```diff
@@ -144,7 +141,7 @@
             }
         }
     
-    public SOAPEnvelope issue(RahasData data) throws TrustException {
+    public SOAPEnvelope issue(RahasData data) {
```

Comment:
Any specific reason to remove this throws statement? Places that use the issue method might be giving compile errors.

Reply:
Thinking retrospectively, I believe I deleted this due to the fact that there's no specific catch block for a TrustException within the method, followed by that I hadn't got any errors on the interpreter level. Your point is very valid, although, since this is a public method, it could be causing issues in a dependant project.